editor/src/java/com/defold/editor/luart/DefoldUserdata.java

```diff
@@ -29,6 +29,6 @@ public DefoldUserdata(Object obj, Object stringRepresentation) {
 
     @Override
     public String tojstring() {
-        return stringRepresentation + ": " + Integer.toHexString(Objects.hashCode(stringRepresentation));
+        return stringRepresentation + ": " + Integer.toHexString(Objects.hashCode(userdata()));
```

Comment:
Just fixed a minor boo-boo: the hash is supposed to help differentiate various user data objects.